dask/dataframe/tests/test_dataframe.py

```diff
@@ -4255,7 +4255,7 @@ def test_idxmaxmin(idx, skipna):
 
     # https://github.com/pandas-dev/pandas/issues/43587
     check_dtype = not all(
-        (_compat.PANDAS_GT_133, skipna is False, isinstance(idx, pd.DatetimeIndex))
+        (_compat.PANDAS_GE_133, skipna is False, isinstance(idx, pd.DatetimeIndex))
```

Comment:
```suggestion
        (PANDAS_GE_133, skipna is False, isinstance(idx, pd.DatetimeIndex))
```
plus import in header